app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonActivityPresenter.kt

```diff
@@ -60,4 +135,17 @@ class ResumeLessonActivityPresenter @Inject constructor(
         R.id.resume_lesson_fragment_placeholder
       ) as ResumeLessonFragment?
   }
+
+  /** Changes the reading text size to normal onBacKPressed. */
+  fun onBackPressed() {
+    setReadingTextSizeNormal()
+  }
```

Comment:
Just directly call this `setReadingTextSizeNormal()` in the activity since it is already public and wrapping it in another function is redundant.

Reply:
Okay.